src/libstore/build/local-derivation-goal.cc

```diff
@@ -502,6 +495,9 @@ void LocalDerivationGoal::startBuilder()
         }
     }
 
+    useUidRange = parsedDrv->getRequiredSystemFeatures().count("uid-range");
+    useSystemdCgroup = parsedDrv->getRequiredSystemFeatures().count("Systemd-cgroup");
```

Comment:
I have the funny feeling this shouldn't be capitalized?

```suggestion
    useSystemdCgroup = parsedDrv->getRequiredSystemFeatures().count("systemd-cgroup");
```